Public/Sdk/Public/Deployment/definition.dsc

```diff
@@ -53,7 +53,7 @@ export interface DeployedFileWithProvenance {
 @@public
 export interface FlattenedResult {
     flattenedFiles: Map<RelativePath, DeployedFileWithProvenance>,
-    flattenedOpaques: Map<RelativePath, OpaqueDirectory>,
+    flattenedOpaques: Map<RelativePath, [OpaqueDirectory, RelativePath]>,  // Tuple of (1) an opaque directory, and (2) a path relative to that opaque directory designating a subdirectory to be added to the deployment

```

Comment:
>[OpaqueDirectory, RelativePath] [](start = 40, length = 31)

I'd rather see a proper named interface for this. Using a tuple for local usage is fine IMO, but this definition makes expressions like [0][1], [0][0] and similar pop up all over the place, which are somewhat cryptic to follow